ADO.NET/AdoOauthSample/OAuthSampleApp.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Vertica.Data.VerticaClient;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using System.Configuration;
+
+internal class AdoOauthSample
+{
+    static void Main(string[] args)
+    {
+        VerticaConnectionStringBuilder vcsb = new VerticaConnectionStringBuilder();
+        vcsb.ConnectionString = ConfigurationManager.ConnectionStrings["ConnectionString"].ConnectionString;
+
+      //  vcsb.OAuthJsonConfig = GetTokenRefreshJsonConfig();

```

Comment:
This is commented out because it is part of token refresh and therefore not in 24.1. Uncommenting it out would make it not compile. The sample is a work in progress.